website/docs/r/vapp_nat_rules.html.markdown

```diff
@@ -0,0 +1,163 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_vapp_nat_rules"
+sidebar_current: "docs-vcd-resource-vapp-nat-rules"
+description: |-
+  Provides a vCloud Director vApp NAT resource. This can be used to create, modify, and delete NAT rules.
+---
+
+# vcd\_vapp\_nat\_rules
+
+Provides a vCloud Director vApp NAT resource. This can be used to create,
+modify, and delete NAT rules in a [vApp network](/docs/providers/vcd/r/vapp_network.html).
+
+!> **Warning:** Using this resource overrides any existing NAT rules on vApp network. It's recommended to have only one resource per vApp and vApp network. 
+
+## Example Usage
+
+```hcl
+resource "vcd_vapp" "web" {
+  name = "web"
+}
+​
+resource "vcd_vapp_network" "vapp-net" {
+  name               = "my-vapp-net"
+  vapp_name          = vcd_vapp.web.name
+  org_network_name   = "my-vdc-int-net"
+  gateway            = "192.168.2.1"
+  netmask            = "255.255.255.0"
+  dns1               = "192.168.2.1"
+
+  firewall_enabled = true
+  nat_enabled      = true
+​
+  static_ip_pool {
+    start_address = "192.168.2.51"
+    end_address   = "192.168.2.100"
+  }
+}
+
+resource "vcd_vapp_org_network" "vapp_org_net" {
+  vapp_name        = vcd_vapp.web.name
+  org_network_name = vcd_network_routed.network_routed.name
+  is_fenced        = true
+}
+
+resource "vcd_vapp_nat_rules" "vapp_nat" {
```

Comment:
Could you change all the **names** in this example to use dashes? It's easier to not make mistakes when names use dashes as opposed to underscores, which are used by resource types. Like this:

```suggestion
resource "vcd_vapp_nat_rules" "vapp-nat" {
```

Then the imaginary `network_routed` to `network-routed`, etc.

Also, use lower case letters in `Vm1` -> `vm1`, etc.

Reply:
changed